lib/manageiq/release/internationalization.rb

```diff
@@ -0,0 +1,103 @@
+module ManageIQ
+  module Release
+    class Internationalization
+
+      attr_reader :branch, :dry_run
+
+      def initialize(branch, dry_run: true)
+        @branch  = branch || 'master'
+        @dry_run = dry_run
+      end
+
+      def execute(command, description = nil)
+        description ||= "*** Running `#{command}` ***"
+        puts description
+        system(command)
+      end
+
+      def create_database_yml
+        puts "*** Creating config/database.yml ***"
+        database_config = YAML.load File.read("config/database.pg.yml")
+        unless database_config.key?('i18n')
+          database_config['i18n'] = database_config['test'].dup
+          database_config['i18n']['database'] = "vmdb_i18n"
+        end
+        File.open("config/database.yml", "w") { |file| file.write(database_config.to_yaml) }
+      end
+
+      def manageiq_pot_filename
+        "locale/manageiq.pot"
+      end
+
+      def manageiq_pot_file_needs_commit?
+        needs_commit = false
+        diffs = git_diff_stats
+        return false if diffs.length != 1
+        return false unless diffs.key?(manageiq_pot_filename)
+        stats = diffs[manageiq_pot_filename]
+        puts "+#{stats[:insertions]}, -#{stats[:deletions]} for #{manageiq_pot_filename}"
+          # If the only changes are to POT-Creation-Date and PO-Revision-Date,
+          # then there are no substantive changes
+          # Hence why this code checks for precisely 2 insertions and deletions
+        (stats[:insertions] != 2) || (stats[:deletions] != 2)
+      end
+
+      def git_diff_stats
+        MiniGit::Capturing.diff(:numstat => true).split("\n").each_with_object({}) do |diff, stats|
```

Comment:
It might be better to use the existing git handle that's part of the Repo object  (e.g. `manageiq_repo.git.capturing.diff`.